src/App.tsx

```diff
@@ -7,16 +7,26 @@ import { ListItem } from "./types/ListItem";
 
 const App = () => {
   const submitForm = () => {
-    setItems([...items, { id: 2, url: url, date: "dfdd" }]);
+    setItems([
+      ...items,
+      {
+        id: items.length + 1,
+        url: url,
+        date: new Date().toLocaleString(),
```

Comment:
I think it's better to save date in iso format rather than storing it already formatted. You can format it when you need to display it. The opposite operation might be a bit more difficult